srcpkgs/zzzy/template

```diff
@@ -1,13 +1,13 @@
 # Template file for 'zzzy'
 pkgname=zzzy
 version=1
-revision=1
+revision=2
```

Comment:
you don't have to increase revision if nothing really changed

Reply:
I changed the package metadata (`homepage`). Isn't this a reason for revbumping?

Reply:
That's hard to say, technically yes, but not sure if it's worth the resources, noone really looks at homepage anyway.